rules/Proj0025.md

```diff
@@ -0,0 +1,33 @@
+# Proj0025: Migrate from ruleset file to .editorconfig file
+Microsoft depricated the of `.ruleset`'s in favour of .editorconfig files.
```

Comment:
"depricated the of '.rulesets'..."
Should be
"depricated the use of '.rulesets'..."
or just "depricated '.rulesets'..."

Reply:
deprecated ;)